### PR TITLE
feat(cloudmon): fix bingocloud monitor data empty

### PR DESCRIPTION
**Pull request by Davidzkeng**
**What this PR does / why we need it**:
fix bingocloud monitor data empty

**Does this PR need to be backport to the previous release branch?**:

NONE

/cc @ioito 
/area cloudmon

### CODE REVIEW

File: pkg/cloudmon/resources/resources.go [hunk 1]
```diff
@@ -586,7 +586,7 @@ func (self *SResources) CollectMetrics(ctx context.Context, userCred mcclient.To
 	resources := self.Cloudproviders.getResources("")
 	cloudproviders := map[string]api.CloudproviderDetails{}
 	jsonutils.Update(&cloudproviders, resources)
-	sh, _ := time.LoadLocation("Asia/Shanghai")
+	sh, _ := time.LoadLocation("UTC")
```
Comment (ioito) [inline pkg/cloudmon/resources/resources.go:589]:
```suggestion
	sh, _ := time.LoadLocation(options.Options.TimeZone)
```

Comment (Davidzkeng) [reply pkg/cloudmon/resources/resources.go:589]: 品高云那边接口有关时间的参数都是使用的UTC时区，这里使用配置会不会有影响

Comment (ioito) [reply pkg/cloudmon/resources/resources.go:589]: 那就在品高云那边time.UTC()转换下

**Comment by ioito** [comment]
close by https://github.com/yunionio/cloudpods/pull/16360

**Pull request closed without merging**